CRM/Contact/Import/Form/Summary.php

```diff
@@ -46,17 +47,11 @@ public function preProcess() {
       $this->assign('dupeError', TRUE);
     }
 
-    $this->assign('groupAdditions', $this->getUserJob()['metadata']['summary_info']['groups']);
-    $this->assign('tagAdditions', $this->getUserJob()['metadata']['summary_info']['tags']);
+    $this->assign('groupAdditions', $this->getUserJob()['metadata']['summary_info']['groups'] ?? []);
+    $this->assign('tagAdditions', $this->getUserJob()['metadata']['summary_info']['tags'] ?? []);
     $this->assignOutputURLs();
     $session = CRM_Core_Session::singleton();
     $session->pushUserContext(CRM_Utils_System::url('civicrm/import/contact', 'reset=1'));
   }
 
-  /**
-   * Clean up the import table we used.
-   */
-  public function postProcess() {
```

Comment:
this function used to do cleanup -now it doesn't. It was never about blocking higher up functions